src/components/gui/gui.jsx

```diff
@@ -49,9 +49,11 @@ const GUIComponent = props => {
                     <Box
                         width={250}
                     >
-                        <TabSelected>
+                        <Tab
+                            selected={true}
```

Comment:
..and of course, you'd have to make it interactive, as well, but I'm not a Professional React Developer who would know how to (really) do that.